src/test/java/com/hocs/test/glue/DataInputStepDefs.java

```diff
@@ -146,26 +146,103 @@ public void iSetTheCorrespondenceChannelTo(String channel) {
 
     @When("^I select to add a correspondent that \"([^\"]*)\" a member of parliament$")
     public void addACorrespondentThatIsOrIsNotAnMP(String isOrIsNot) {
-        dataInput.clickAddCorrespondentLink();
+        page.clickOn(dataInput.addCorrespondentLink);
+
 
         if (isOrIsNot.toUpperCase().equals("IS")) {
-            dataInput.clickCorrespondentIsAMember();
+            page.clickOn(dataInput.correspondentMemberYesRadioButton);
         } else if (isOrIsNot.toUpperCase().equals("IS NOT")) {
-            dataInput.clickCorrespondentIsNotAMember();
+            page.clickOn(dataInput.correspondentMemberNoRadioButton);
         }
+    }
 
+    @When("^they complete the first data input screen$")
+    public void completeFirstDataInputScreen() {
+        dataInput.enterDayOfCorrespondenceSent("01");
```

Comment:
There is a method for this on the dataInput page class

Reply:
And some of the below as well, take it into the dataInput class and try to use one method in the stepDefinitions